tool_quality_checks/fix_import_file.py

```diff
@@ -248,7 +252,7 @@ def fix_vector_layer(in_layer, layer_name="", epsg=3857):
             break
 
     logger.info("check  - Features count")
-    out_feature_count = out_layer.GetFeatureCount()
+    out_feature_count = out_layer.__len__()
```

Comment:
Dit is normaliter hetzelfde als:

```suggestion
    out_feature_count = len(out_layer)
```

Die dubbele-underscore methodes moet je normaliter niet zelf aanroepen. Meestal zijn het methodes die door standaard python zaken gebruikt worden. `str(iets)` roept achter de schermen effectief `iets.__str__()` aan, bijvoorbeeld. Idem dus met `len()`.